docs/integrations/flutter-specific-workflow-steps/flutter-test.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Flutter Test
+metaTitle: Flutter Test
+metaDescription: Flutter Test
+sidebar_position: 4
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Flutter Test
+
+This component allows you to run [**Flutter Unit Tests**](https://docs.flutter.dev/cookbook/testing/unit/introduction#run-tests-in-a-terminal). Please note that it requires the preceding [**Flutter Install**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install) and [**Git Clone**](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) steps to run.
+
+
+### Prerequisites
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [**Git Clone**](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) | This step will clone your project through the connected git provider and create the `$AC_REPOSITORY_DIR` variable. |
+| [**Flutter Install**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install) | This step will install the Flutter SDK. If a version is not specified, it will install the latest **stable** version. |
```

Comment:
This information can also be added:
```suggestion
| [**Flutter Install**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install) | This step will install the [Flutter SDK version](https://flutter-ko.dev/development/tools/sdk/releases). If a version is not specified, it will install the latest **stable** version. |
```